packages/slappforge-sdk-redis/lib/commands/CommandsHandler.js

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2017-2018 SLAppForge Lanka Private Ltd. (https://www.slappforge.com).
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @author Lahiru Ananda
+ */
+
+let async = require('async');
+let connectionManager = require('../ConnectionManager');
+
+module.exports = function () {
+
+    this.execute = function (command, callback) {
+        let params, client, counter, errCount = 0;
+        let responseArray = [];
+        try {
+            params = command.params;
+            counter = params.length;
+            async.forEach(params, (param, callback) => {
+                handle(
+                    command.type,
+                    command.operation,
+                    command.clusterSpec,
+                    param,
+                    (response, redisClient) => {
+                        counter--;
+                        if (response.error)
+                            errCount++;
+                        redisClient && counter === 0 ? client = redisClient : redisClient.quit();
```

Comment:
Could lead to `quit()` getting invoked on an undefined variable (in case `redisClient` happens to be undefined)